src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/http_client.html.twig

```diff
@@ -121,11 +124,28 @@
                                     </tr>
                                     </tbody>
                                 </table>
+                                {# tmp: introspect the "trace" #}
+                                {{ dump(trace) }}
                             {% endfor %}
                         {% endif %}
                     </div>
                 </div>
             {% endfor %}
         {% endif %}
     </div>
+
+    <script type="text/javascript">//<![CDATA[
+
+        document.querySelectorAll('[data-clipboard-text]').forEach(function(button) {
+            button.addEventListener('click', function() {
+                navigator.clipboard.writeText(button.getAttribute('data-clipboard-text'));
```

Comment:
instead of writing support for `Copy to clipboard` as an inline script in this panel, I suggest adding such a feature in the shared JS of the profiler pages, so that it can be reused by any other panel too (as we do for tabs and toggles).

Reply:
I'll refactor indeed